### PR TITLE
Use ExternalLink to automatically use noopener/noreferrer

### PR DESCRIPTION
**Pull request by ariya**
This is to tweak the version update message so as to resolve the code scanning warning: https://github.com/metabase/metabase/security/code-scanning/11

### CODE REVIEW

**Review by gusaiani — APPROVED** [review]
Let me dare an approval here.

Caveats:

Did not see where this link is rendered in the application.

Was not clear to me how `dataMetabaseEvent` goes all the way into markup as `data-metabase-event`.

**Review by daltojohnso — APPROVED** [review]

File: frontend/src/metabase/admin/settings/components/SettingsUpdatesForm.jsx [hunk 1]
```diff
@@ -54,7 +55,7 @@ export default class SettingsUpdatesForm extends Component {
               target="_blank"
```
Comment (paulrosenzweig) [inline frontend/src/metabase/admin/settings/components/SettingsUpdatesForm.jsx:55]: ExternalLink will set target based on whether the host matches. You could drop this without changing behavior.

**Comment by ariya** [comment]
> Was not clear to me how `dataMetabaseEvent` goes all the way into markup as `data-metabase-event`.

Good catch, thank you @gusaiani.

@nemanjaglumac I think we ought to fix this in master, too.

File: frontend/src/metabase/admin/settings/components/SettingsUpdatesForm.jsx (hunk 1 above)
Comment (ariya) [reply frontend/src/metabase/admin/settings/components/SettingsUpdatesForm.jsx]: Good tip, thanks @paulrosenzweig. I've adjusted it accordingly.

**Comment by nemanjaglumac** [comment]
> Was not clear to me how `dataMetabaseEvent` goes all the way into markup as `data-metabase-event`.
> @nemanjaglumac I think we ought to fix this in master, too.

@ariya @gusaiani Just to make sure we're all on the same page.
I remember being confused about this when I made the changes. So, to clarify something first:

- `<Link>` and pure html anchor `<a>` are the ones with `data-metabase-event` being set directly.
- `<ExternalLink>` OTOH is the one where I needed to add `dataMetabaseEvent` as a prop so [this line](https://github.com/metabase/metabase/pull/16111/files#diff-e364fbcb4aee74a0877769347f6df51435c4b99519f0601be95481f4157c0d87R46) might actually not work. 

Check out the ExternalLink component on master:
https://github.com/metabase/metabase/blob/master/frontend/src/metabase/components/ExternalLink.jsx#L17

**Comment by gusaiani** [comment]
@nemanjaglumac, I will defer to @ariya and others with better context than me on this one.

**Comment by ariya** [comment]
> * `<ExternalLink>` OTOH is the one where I needed to add `dataMetabaseEvent` as a prop so [this line](https://github.com/metabase/metabase/pull/16111/files#diff-e364fbcb4aee74a0877769347f6df51435c4b99519f0601be95481f4157c0d87R46) might actually not work.

It should work, since `{...props}` should "spread" the props, propagating them from `ExternalLink` to `a`:

image


> Check out the ExternalLink component on master:

ExternalLink starts to diverge between 39 and master. Probably wise to unify it to avoid upcoming merge conflict.

**Comment by nemanjaglumac** [comment]
Thanks @ariya. I will update it on `master`.

**Pull request merged**